src/main/java/com/spikes2212/util/Limelight.java

```diff
@@ -0,0 +1,73 @@
+package com.spikes2212.util;
+
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableInstance;
+
+public class Limelight {
+
+    public enum LedMode {
+
+        BY_PIPELINE(0),
+        OFF(1),
+        BLINK(2),
+        ON(3);
+
+        int state;
+
+        LedMode(int state) {
+            this.state = state;
+        }
+
+        int getState() {
+            return state;
+        }
+    }
+
+    public enum CameraMode {
+        VISION_PROCESSING(0),
+        DRIVERS(1);
+
+        int mode;
+
+        CameraMode(int mode) {
+            this.mode = mode;
+        }
+
+        int getMode() {
+            return mode;
+        }
+    }
+
+    private NetworkTable networkTable;
+
+    public Limelight() {
```

Comment:
receive string for limelight network table with default option `limelight` - usefull for cases when more than one limelight is used.